spec.html

```diff
@@ -602,6 +602,17 @@ <h1>Iterator.prototype.toArray ( )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-iteratorprototype.toasync">
+        <h1>Iterator.prototype.toAsync ( )</h1>
+        <emu-alg>
+          1. Let _syncIteratorRecord_ be ? GetIteratorDirect(*this* value).
```

Comment:
maybe hot take
`1. Return ? Call(%AsyncIterator.from%, %AsyncIterator%, &laquo; *this* value &raquo;).`